.gitignore

```diff
@@ -1,6 +1,4 @@
 node_modules
 coverage
-todo.txt
```

Comment:
this file doesn't seem to be in the repo anymore but we might as well keep it  in `.gitignore` just in case it gets added again and we want it ignored automatically?

Reply:
Done!